docs/setup-cluster/deploy-cluster/slurm/upgrade-on-hpc.rst

```diff
@@ -0,0 +1,39 @@
+.. _upgrade-on-hpc:
+
+#################################
+ Upgrade Determined on Slurm/PBS
+#################################
+
+This procedure describes how to upgrade Determined on an HPC cluster managed by the Slurm or PBS
+workload managers. Use this procedure when an earlier version of Determined is installed,
+configured, and functioning properly.
```

Comment:
.. _upgrade-on-hpc:

#################################
 Upgrade Determined on Slurm/PBS
#################################

These steps describe how to upgrade Determined on an HPC cluster that is managed by the Slurm or PBS
workload managers. You can follow these steps when an earlier version of Determined is installed,
configured, and functioning properly.